src/constants.ts

```diff
@@ -20,3 +20,9 @@ export const INPUT_CACHE = 'cache';
 export const INPUT_JOB_STATUS = 'job-status';
 
 export const STATE_GPG_PRIVATE_KEY_FINGERPRINT = 'gpg-private-key-fingerprint';
+
+export const M2_DIR = '.m2';
+export const MVN_SETTINGS_FILE = 'settings.xml';
+export const MVN_TOOLCHAINS_FILE = 'toolchains.xml';
+export const INPUT_MVN_TOOLCHAIN = 'mvn-toolchain';
+export const INPUT_MVN_TOOLCHAIN_ID = 'mvn-toolchain-id';
```

Comment:
These new inputs need to be added to the [action.yml](https://github.com/actions/setup-java/blob/5b36705a13905facb447b6812d613a06a07e371d/action.yml#L5).